src/main/java/duke/utils/Input.java

```diff
@@ -0,0 +1,159 @@
+package duke.utils;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import duke.exception.PromptCannotBeEmptyException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Input {
+    static Scanner sc = new Scanner(System.in);
```

Comment:
Maybe you can consider a better naming for `sc`, one that readers can understand more easily.